security/src/test/scala/pl/touk/nussknacker/ui/security/basicauth/BasicAuthenticationSpec.scala

```diff
@@ -46,32 +46,12 @@ class BasicAuthenticationSpec extends AnyFunSpec with Matchers with ScalatestRou
     }
   )
 
-  it("should permit an anonymous user") {
```

Comment:
why did we remove the tests?

Reply:
I wanted to move some tests regaring anonymous access to the `AuthenticationManager` since it's now its resposibility to handle it. However i did it a bit too heavy handed and removed two tests that should still be in the `BasicAuthenticationSpec` - I added them back.